File: rss-reader/app/src/main/java/com/abubusoft/kripton/examples/rssreader/api/BbcApi.java
package com.abubusoft.kripton.examples.rssreader.api;

import com.abubusoft.kripton.examples.rssreader.model.RSSFeed;

import retrofit2.Call;
import retrofit2.http.GET;

public interface BbcApi {
    @GET("news/rss.xml?edition=int")
    Call<RSSFeed> loadRSSFeed();
}

File: kotlin-rss-reader/app/src/main/java/com/abubusoft/kripton/example/rssreader/service/api/BbcApi.kt
package com.abubusoft.kripton.example.rssreader.service.api


import com.abubusoft.kripton.example.rssreader.service.model.RssFeed
import retrofit2.Call
import retrofit2.http.GET

interface BbcApi {
    @GET("news/rss.xml?edition=int")
    fun loadRSSFeed(): Call<RssFeed>
}